RR.App.Tests/Controllers/HRIS/PropertyAccessControllerUnitTests.cs

```diff
@@ -134,80 +127,28 @@ public async Task UpdatePropertyRoleAccessReturnsFail()
         _propertyAccessMockService.Verify(service => service.UpdatePropertyAccess(0, PropertyAccessLevel.read), Times.Once);
     }
 
-    [Fact]
-    public async Task GetUserIdReturnsOkResult()
-    {
-        var email = "test@retrorabbit.co.za";
-        var employeeId = 1;
-        var authUserId = "authUser123";
-
-        var employee = new EmployeeDto
-        {
-            Id = employeeId,
-            AuthUserId = null
-        };
-
-        var claims = new List<Claim>
-        {
-        new Claim(ClaimTypes.NameIdentifier, authUserId)
-        };
-        var identity = new ClaimsIdentity(claims, "TestAuthType");
-        var claimsPrincipal = new ClaimsPrincipal(identity);
-
-        _employeeMockService.Setup(service => service.GetEmployee(email))
-                            .ReturnsAsync(employee);
-
-        var httpContext = new DefaultHttpContext
-        {
-            User = claimsPrincipal
-        };
-        _propertyAccessController.ControllerContext = new ControllerContext
-        {
-            HttpContext = httpContext
-        };
-
-        var result = await _propertyAccessController.GetUserId(email);
-
-        var okResult = Assert.IsType<OkObjectResult>(result);
-        Assert.Equal(employeeId, okResult.Value);
-
-        _employeeMockService.Verify(service => service.GetEmployee(email), Times.Once);
-
-        _employeeMockService.Verify(service => service.UpdateEmployee(It.Is<EmployeeDto>(e =>
-            e.Id == employeeId && e.AuthUserId == authUserId), email), Times.Once);
-    }
-
-    [Fact]
-    public async Task GetUserIdReturnsNotFoundResultWhenExceptionThrown()
-    {
-        var email = "test@retrorabbit.co.za";
+    //[Fact]
```

Comment:
```suggestion
    [Fact(Ignore="Test needs updating")]
```

Reply:
I would rather just mark the test to be ignored. Rather than creating ghost code :)

Reply:
@Wimmas please fix